go_node_engine/requests/NetManagerRequests.go

```diff
@@ -22,6 +22,14 @@ type connectNetworkRequest struct {
 	PortMappings   string `json:"portMappings"`
 }
 
+type connectNetworkRequestUnikernel struct {
+	Pid            int    `json:"pid"`
+	Servicename    string `json:"serviceName"`
+	Instancenumber int    `json:"instanceNumber"`
+	PortMappings   string `json:"portMappings"`
+	IsUnikernel    bool   `json:"isUnikernel"`
```

Comment:
Do we need the bool Is Unikernel? In the net component, you already declared two different endpoints, right?

Reply:
No not anymore. I will remove it.